.github/workflows/deploy.yml

```diff
@@ -72,6 +82,30 @@ jobs:
           azure_credentials: ${{ secrets.AZURE_CREDENTIALS }}
           terraform_vars: workspace_variables/dev.tfvars.json
 
+  deploy_review:
+    name: Deploy to review environment
+    concurrency: deploy_review_${{ github.event.pull_request.number }}
+    needs: [docker]
+    runs-on: ubuntu-latest
+    if: contains(github.event.pull_request.labels.*.name, 'deploy') || github.event_name != 'pull_request'
+    environment:
+      name: review
+      url: ${{ steps.deploy.outputs.environment_url }}
```

Comment:
This will update the URL of the "review" environment which is shared between all the review apps, so the dev may not be able to see their app URL.
Can we remove it from here and display it as a PR comment instead? See: https://github.com/DFE-Digital/teaching-vacancies/blob/main/.github/workflows/build_and_deploy.yml#L266